indico/modules/events/papers/forms.py

```diff
@@ -88,6 +88,7 @@ class PaperReviewingSettingsForm(IndicoForm):
     announcement = IndicoMarkdownField(_('Announcement'), editor=True)
     scale_lower = IntegerField(_('Scale (from)'), [InputRequired()])
     scale_upper = IntegerField(_('Scale (to)'), [InputRequired()])
+    hide_peer_reviewed_papers = BooleanField(_('Keep papers hidden'), widget=SwitchWidget(), description=_("Hides papers from participants that have been marked as 'accepted'."))
```

Comment:
- Wrap the description into a new line.
- How about "Keep papers hidden from participants even after they have been accepted"? I think that makes it clearer - right now it sounds like the paper would suddenly get hidden once accepted..